data/src/test/java/org/apache/iceberg/TestMergingMetrics.java

```diff
@@ -192,16 +187,16 @@ public void verifyRandomlyGeneratedRecordsMetric() throws Exception {
         .map(Types.NestedField::fieldId)
         .forEach(
             id ->
-                Assert.assertNull(
-                    "NaN count for field %s should be null", metrics.nanValueCounts().get(id)));
+                assertThat(metrics.nanValueCounts().get(id))
+                    .as("NaN count for field %s should be null")
+                    .isNull());
   }
 
   private void assertNaNCountMatch(
       Long expected, Map<Integer, Long> nanValueCount, Types.NestedField field) {
-    Assert.assertEquals(
-        String.format("NaN count for field %s does not match expected", field.name()),
-        expected,
-        nanValueCount.get(FIELDS_WITH_NAN_COUNT_TO_ID.get(field)));
+    assertThat(nanValueCount.get(FIELDS_WITH_NAN_COUNT_TO_ID.get(field)))
```

Comment:
since this is checking something in a map it would be better to adjust this to `assertThat(nanValueCount).containsEntry(..., expected)`

Reply:
cool, thx, done